class-vip-support-user.php

```diff
@@ -654,7 +661,7 @@ public static function is_verified_automattician( $user_id ) {
 		return ( $is_a8c_email && $email_verified );
 	}
 
-	public function user_has_vip_support_role( $user_id, $active_role = true ) {
+	public static function user_has_vip_support_role( $user_id, $active_role = true ) {
```

Comment:
Not a big deal but feels somewhat inconsistent to make this static but still have a method like `user_has_verified_email` which is not static.

Reply:
Yeah, it's a bit strange, but the method doesn't do anything with the instance, and this was needed to access it from the new `remove()` method. A quick glance indicates that most of the public methods could be made static.